compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -1259,15 +1255,6 @@ impl<'a> Parser<'a> {
         maybe_whole!(self, NtVis, |x| x.into_inner());
 
         self.expected_tokens.push(TokenType::Keyword(kw::Crate));
```

Comment:
I'm not sure on whether this line should be removed.

Reply:
UI tests and `rustc_parse` tests both pass with this removed, so I don't think it's necessary. Removing.